integration/tests/access/unstaked_node_test.go

```diff
@@ -40,23 +40,18 @@ func TestUnstakedAccessSuite(t *testing.T) {
 }
 
 func (suite *UnstakedAccessSuite) TearDownTest() {
-	// avoid nil pointer errors for skipped tests
-	if suite.cancel != nil {
-		defer suite.cancel()
-	}
-	if suite.net != nil {
```

Comment:
the test does not set `suite.net` to anything. Instead it calls `suite.net.Start` and `suite.net.Remove`. Hence, [similar to the MVP test ](https://github.com/onflow/flow-go/blob/master/integration/tests/common/mvp_test.go#L33-#L34) it should also do the same.

Reply:
Not sure I understand? Suite.net is set, in the buildNetworkConfig function, is it not?

Reply:
yes, it is set in `buildNetworkConfig`.